src/Components/Components/CardSerie/CardSerie.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+//importacion de librerias externas
+import { Card } from "react-bootstrap";
+
+/**
+ *
+ * @param {*} param0
+ */
+export default function CardSerie({ title = "titulo", imageURL }) {
```

Comment:
Avoid default arguments while using React components. Use default props instead.

Reply:
Also put the export deafult statement at the end of the file.

export default CardSerie;

Reply:
Usa proptypes, si o si. Acostumbrate a eso de una vez